src/trix/models/html_parser.coffee

```diff
@@ -64,7 +64,6 @@ class Trix.HTMLParser extends Trix.BasicObject
   appendBlockForTextNode: (node) ->
     element = node.parentNode
     return if element is @currentBlockElement
-    return unless @isBlockElement(element)
```

Comment:
> Removing the call to `isBlockElement` fixes the issue and all the other tests still pass. If the `isBlockElement` check is required for other use cases this issue may be solved by bypassing the `isBlockElement` check if the TextNode's parent element in the trix-editor.

We have pretty good test coverage, but you never know! I'd prefer to go the safer route and check the parent element. Example:
```coffee
    return unless @isBlockElement(element) or element is @containerElement
````